src/components/SplitpanesBar.vue

```diff
@@ -100,6 +139,29 @@ export default {
     },
     activeView: function() {
       return store.state.splitFlow.activeView;
+    },
+    envVars: function () {
+      return {
+        API_LOCATION: store.state.settings.sparcApi,
+        ALGOLIA_INDEX: store.state.settings.algoliaIndex,
+        ALGOLIA_KEY: store.state.settings.algoliaKey,
+        ALGOLIA_ID: store.state.settings.algoliaId,
+        PENNSIEVE_API_LOCATION: store.state.settings.pennsieveApi,
+        NL_LINK_PREFIX: store.state.settings.nlLinkPrefix,
+        ROOT_URL: store.state.settings.rootUrl,
+      };
+    },
+    boundary: function () {
+      let b = document.querySelector(".tab-container")
+      return b;
+    },
```

Comment:
The tab-container class is now named container after the change get reverted.

By the way, it is probably better to add a @hook:mounted method in the SplitpaneBar components of the template in splitDialog which will then pass in the boundary into it.